bin/run_mtl_loop

```diff
@@ -38,6 +40,9 @@ if ns.obscon == "BACKUP" and ns.survey == "main":
     raise RuntimeError(msg)
     log.error(msg)
 
+if not ns.noarchivecheck:
+    check_archiving(zcatdir=ns.zcatdir, mtldir=ns.mtldir)
```

Comment:
for consistency, would it make sense to propagate in the call to `check_archiving()` the `ns.survey` and `ns.obscon` argument?
for instance if `run_mtl_loop` is called for main/bright, then the check is done for main/bright only?

I am aware that cases other than main/bright and main/dark may never be used -- and `check_archiving()` would be irrelevant for those; so maybe you can just request in `check_archiving()` that only those cases are valid?

Reply:
I think this is a smart idea. 

I added a required `obscon` argument and a `survey=survey` kwarg to `check_archiving()` and made the appropriate changes to propagate these arguments throughout the function.

Reply:
Also, `run_mtl_loop` now additionally passes `obscon` and `survey` to the `check_archiving()` function.